src/Shared/RoslynUtils/ParsabilityHelper.cs

```diff
@@ -155,46 +163,54 @@ internal static Bindability GetBindability(ITypeSymbol typeSymbol, WellKnownType
     {
         bindabilityMethod = null;
         bindMethodSymbol = null;
+        IMethodSymbol? bindAsyncMethod = null;
 
-        if (IsBindableViaIBindableFromHttpContext(typeSymbol, wellKnownTypes))
-        {
-            bindabilityMethod = BindabilityMethod.IBindableFromHttpContext;
-            return Bindability.Bindable;
-        }
-
-        // TODO: Search interfaces too. See MyBindAsyncFromInterfaceRecord test as an example.
```

Comment:
TODO not applicable anymore?

Answer: Looks like it is supported already: https://github.com/dotnet/aspnetcore/blob/5bdb8b46da61176498d4b43f17759cd8cdc41be5/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.BindAsync.cs#L37

Reply:
Yeah, I resolved this TODO a while back in an unrelated PR and didn't remove the comment. We do support this scenario as you mentioned.